app/controllers/verify/finance_controller.rb

```diff
@@ -26,6 +27,20 @@ def create
 
     private
 
+    def submit_idv_form
+      result = idv_finance_form.submit(step_params)
+      analytics.track_event(Analytics::IDV_FINANCE_CONFIRMATION, result.to_h)
+
+      return if result.success?
+
+      render_failure
+      render_form
+    end
+
+    def form_valid_but_vendor_validation_failed?
```

Comment:
Well each form has a different name (ex `idv_profile_form`, `idv_phone_form`, `idv_finance_form`) ... so I guess I could just rename those to `idv_form` and then make it consistent again

Thanks for pointing this out, I'm fixing it now